lib/15_in_words.rb

```diff
@@ -0,0 +1,75 @@
+class Fixnum
+  def in_words
+    hash = {
+      0 => nil,
+      1 => "one",
+      2 => "two",
+      3 => "three",
+      4 => "four",
+      5 => "five",
+      6 => "six",
+      7 => "seven",
+      8 => "eight",
+      9 => "nine",
+      10 => "ten",
+      11 => "eleven",
+      12 => "twelve",
+      13 => "thirteen",
+      14 => "fourteen",
+      15 => "fifteen",
+      16 => "sixteen",
+      17 => "seventeen", 
+      18 => "eighteen",
+      19 => "nineteen",
+    }
+
+    ty = {
+      1 => "ten",
+      2 => "twenty",
+      3 => "thirty",
+      4 => "forty",
+      5 => "fifty",
+      6 => "sixty",
+      7 => "seventy",
+      8 => "eighty",
+      9 => "ninety"
+    }
+
+    scale = [
+      nil,
+      "thousand",
+      "million",
+      "billion",
+      "trillion"
+    ]
+
+    chunks = []
+    num = self
+    while num > 0
```

Comment:
You have 3 distinct chunks of logic here, I recommend splitting it into 3 submethods instead of putting it all into 1 method. The 1 method would contain the 3 submethods. For unit testing (which you don't need here) that makes things a lot easier.